integration-tests/FASTN.ftd

```diff
@@ -1,4 +1,9 @@
 -- import: fastn
 
 -- fastn.package: integration-tests
-endpoint: http://127.0.0.1:5000
+
+-- fastn.endpoint: http://fastn.com
+mountpoint: /ftd/
```

Comment:
Let's revers the order, I think this would look better:

```ftd
-- fastn.package: foo.com

-- fastn.mount: /ftd/
forward-to: http://fastn.com
include-prefix: true ;; true is default
```

If `include-prefix` is `true`, `foo.com/ftd/hello` will be forwarded to `http://fastn.com/ftd/hello`, if it was `false`, it will be forwarded to `http://fastn.com/hello`.

Reply:
Or should we put it inside `fastn.url-mappings`](https://github.com/orgs/fastn-stack/discussions/1623)?

```
-- fastn.url-mappings:

/ftd/* -> http://fastn.com/ftd/*
```

In this case how would we differentiate from redirection?

Reply:
Maybe:

```
-- fastn.url-mappings:

/ftd/* -> http+proxy://fastn.com/ftd/*
```